ts_src/psbt.ts

```diff
@@ -1746,49 +1736,110 @@ function randomBytes(options?: RngOpts): Buffer {
   return rng(32);
 }
 
-interface BlindingData {
-  value: string;
-  ag: Buffer;
-  abf: Buffer;
-  vbf: Buffer;
+export interface BlindingData {
+  satoshis: number;
+  amountBlinder: string;
+  asset: string;
+  assetBlinder: string;
 }
 
-function getBlindingDataForInput(
-  prevout: WitnessUtxo,
-  blindPrivKey?: Buffer,
+// Buffer = privateBlindingKey for conf inputs
+// BlindingData = blinders for already unblinded conf inputs
+// undefined = unconfidential inputs
+export type BlindingDataLike = Buffer | BlindingData | undefined;
+
+/**
+ * Compute outputs blinders
+ * @param inputsBlindingData the transaction inputs blinding data
+ * @param outputsData data = [satoshis, asset] of output to blind
+ * @returns an array of BlindingData[] corresponding of blinders to blind outputs specified in outputsData
+ */
+export function computeOutputsBlindingData(
```

Comment:
why exporitng this function?

Reply:
This can be helpful in case we may want to support producing blinders in wallets/UI to be exported, for example we could have the same feature of Green that does this. 

Maybe we could do in other ways, but If I got right would allow for example Marina of using his own cache to produce that

Reply:
Allright, let's move this to confidential then since it's a more appropriate place for such method.
cc @louisinger